sympy/geometry/polygon.py

```diff
@@ -374,6 +374,68 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+
+    def second_moment_of_area(self, p='centroid'):
+        """Returns the second moment and product moment of area of a two dimensional polygon.
+
+        Parameters
+        ==========
+
+        p : Point, two-tuple of sympifiable objects, or None (default=None)
+            p is the point about which second moment of area is to be found.
+            if None, it will be calculated about the centroid of the polygon.
+
+        Notes
+        =====
```

Comment:
I think we can remove this `Notes` section and instead move it where `p` is described.